cli/azd/cmd/infra_create.go

```diff
@@ -106,7 +106,7 @@ func (i *infraCreateAction) Run(ctx context.Context) (*actions.ActionResult, err
 		return nil, err
 	}
 
-	infraManager, err := provisioning.NewManager(ctx, env, prj.Path, prj.Infra, !i.flags.global.NoPrompt)
+	infraManager, err := provisioning.NewManager(ctx, env, prj.Path, prj.Infra, i.console.IsUnformatted())
```

Comment:
I noticed that was missed when I did #905. This has now been intentionally changed so that `--no-prompt` will still print spinner update messages.